Common/AR/Ar.md

```diff
@@ -324,6 +324,10 @@ dependencies {
 4. The project is ready to be built and run on an Android device. You need to verify the compatibility of your device
 with the ARKit.
 
+## Calibration view item
+
+...
```

Comment:
Does this mean documentation is in the works? Or do we need a follow up issue to add this?

Reply:
I added that yesterday evening to be sure to not forget.
Finally, I removed that and added 1 line at the end of "Overview":

> The `CalibrationView` can be used to change manually the location and
heading of the scene, to ensure accuracy between the device live video and the scene.